### PR TITLE
[doc] unhandled vs empty vs ignore

### PR DESCRIPTION
**Pull request by robstoll**
## References

References #27443

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/fsm.md [hunk 1]
```diff
@@ -43,6 +43,17 @@ Scala
 Java
 :  @@snip [FSMSocTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/FSMDocTest.java) { #simple-state}
 
+@scala[
+The method `idle` above makes use of `Behaviors.unhandled` which advises the system to reuse the previous behavior, 
+including the hint that the message has not been handled.
+There are two related behaviors:
+- return `Behaviors.empty` as next behavior in case you reached a state where you don't expect messages any more.
```
Comment (patriknw) [inline akka-docs/src/main/paradox/typed/fsm.md]: I think there must be a newline before bullet list in paradox markdown.

**Review by patriknw — COMMENTED** [review]
I think that ref link to 20984 is wrong ticket?

**Comment by robstoll** [comment]
That's what happens if suddenly the drop down of github shows up on mobile and does not dissappear. I only wanted to insert a 2 infront of the Rest. Fixed it and I will add the new line later on

**Comment by patriknw** [comment]
OK TO TEST

**Review by patriknw — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

**Pull request merged**